bin/octocatalog-diff

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/python3 -u
```

Comment:
Ah, so this was to get unbuffered output since I think I was getting stdout and stderr mixed together (it's been a little while now though so I don't exactly remember). However, given how it's used now, I don't think this is actually necessary for use in Jenkins.